### DISCUSSION TITLE
Is Docker possible für my Synology NAS?

### DISCUSSION THREAD

**Discussion by PaulEins** [streamlink/streamlink-twitch-gui#1024] [Q&A]
Hi, is ther a docker version?

I will install this on my Synology NAS and i will that the tool shwo herself the a new chanel live. Then download or capture the stream. i came back later an dwill see the video. 

Can you help me?

**Comment by bastimeyer** [comment]
https://github.com/streamlink/streamlink-twitch-gui#%E2%84%B9%EF%B8%8F-description
